src/components/Dashboard/Dashboard.tsx

```diff
@@ -1,18 +1,33 @@
 import React from 'react';
 
 import './dashboard.css';
-import MixedChart from './MixedChart/MixedChart';
-import { colors } from './colors';
-import AverageValueDisplay from './AverageValueDisplay/AverageValueDisplay';
-import { AGGREGATE, ATTENTION, MOOD } from './constants';
-import ChartToggler from './ChartToggler/ChartToggler';
+import MixedChart from '../Charts/MixedChart/MixedChart';
+import { colors } from '../../colors';
+import AverageValueDisplay from '../Charts/AverageValueDisplay/AverageValueDisplay';
+import { AGGREGATE, ATTENTION, MOOD } from '../../constants';
+import ChartToggler from '../Charts/ChartToggler/ChartToggler';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { todoType } from '../../types';
 
 const Dashboard = () => {
+
+  const navigate = useNavigate();
+  const location = useLocation();
+  const { accuracy, data }: todoType = location.state;
+
+  const toggleView = () => {
+    navigate('/detailed', { state:{ accuracy, data }});
+  };
+
   return (
-    <div className='container-fluid px-4 mt-3 '>  {/* FLUID? */}
+    <div className='container-fluid px-4 mt-3 pe-5'>
       <div className='row d-flex' style={{ height: 'calc(50vh - 50px)', maxHeight: 500 }}>
-        <div className='d-flex flex-column align-items-center pt-6 col-sm-4 col-md-3 col-lg-2' >
-          <div className='card p-3 py-4 mt-1'>
+        <div className='d-flex flex-column align-items-center col-sm-4 col-md-3 col-lg-2' >
+          <div className="btn-group d-flex mb-2" >
+            <button className={'p-1 py-2 mb-0 btn btn-primary shadow-none toggle-view-btn'} type="button" style={{background: colors.headers}}>Dashboard</button>
+            <button className={'p-1 py-2 mb-0 btn btn-primary shadow-none toggle-view-btn not-selected'} onClick={toggleView} type="button" style={{background: colors.headers}}>Detailed</button>
```

Comment:
I see you use useNavigate instead of <Link> component. It's probably nicer in the jsx because you pass it on click instead of wrapping the button in a link. Though since I used Link in the alert, we should maybe decide for one or the other in the refactoring for consistency? What do you think? Not a talk for now though, it works and that's the most important thing, good job.

Reply:
yeah, for this probably better to use <Link> but i didnt event think of it